src/modules/components/proposal/proposalActions/proposalActions/proposalActions.stories.tsx

```diff
@@ -122,9 +123,46 @@ export const CustomActions: Story = {
                 customActionComponents={{
                     CUSTOM_ACTION: CustomActionComponent,
                 }}
+                emptyStateDescription="No actions added. Consider adding some."
             />
         );
     },
 };
 
+export const EmptyActions: Story = {
+    render: () => {
+        const actions: IProposalAction[] = [];
+
+        return <ProposalActions actions={actions} emptyStateDescription="No actions added. Consider adding some." />;
```

Comment:
Let's use args when we only needs to specify the component props without a custom render logic